sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -233,6 +233,7 @@ class JobStatusResult(object):  # pylint: disable=useless-object-inheritance, to
     :ivar int documents_not_yet_started_count: Number of documents that have not yet started being translated.
     :ivar int documents_cancelled_count: Number of documents that were cancelled for translation.
     :ivar int total_characters_charged: Total characters charged across all documents within the job.
+    :ivar int has_completed: boolean to check whether a document finished translation or not.
```

Comment:
This should be `:ivar bool has_completed:`

I think we'll need to be a bit more explicit about what this is in the docstrings. Something like:
"If the status returned indicates that the translation job has completed. A translation job is considered 'complete' if it has reached a terminal state like 'Succeeded', 'Cancelled', or 'Failed'."

edit: same comment for the DocumentStatusResult.has_completed